<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VentanaPrincipal</class>
 <widget class="QMainWindow" name="VentanaPrincipal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>758</width>
    <height>351</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VentanaPrincipal</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>741</width>
      <height>301</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tNumeros">
     <attribute name="title">
      <string>Numeros</string>
     </attribute>
     <widget class="QToolButton" name="tbLoad">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>240</y>
        <width>28</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>resources/folder_2_open-512.png</normaloff>resources/folder_2_open-512.png</iconset>
      </property>
     </widget>
     <widget class="QToolButton" name="tbRun">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>240</y>
        <width>28</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Load</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>resources/play.png</normaloff>resources/play.png</iconset>
      </property>
     </widget>
     <widget class="QTextBrowser" name="tebLoaded">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>31</y>
        <width>311</width>
        <height>201</height>
       </rect>
      </property>
      <property name="source">
       <url>
        <string>file:///</string>
       </url>
      </property>
     </widget>
     <widget class="QTextBrowser" name="tebModified">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>31</y>
        <width>301</width>
        <height>201</height>
       </rect>
      </property>
      <property name="source">
       <url>
        <string>file:///</string>
       </url>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcdT1">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>30</y>
        <width>61</width>
        <height>23</height>
       </rect>
      </property>
      <property name="smallDecimalPoint">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcdT2">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>70</y>
        <width>61</width>
        <height>23</height>
       </rect>
      </property>
      <property name="smallDecimalPoint">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcdT3">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>110</y>
        <width>61</width>
        <height>23</height>
       </rect>
      </property>
      <property name="smallDecimalPoint">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcdT4">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>150</y>
        <width>61</width>
        <height>23</height>
       </rect>
      </property>
      <property name="smallDecimalPoint">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcdT5">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>190</y>
        <width>61</width>
        <height>23</height>
       </rect>
      </property>
      <property name="smallDecimalPoint">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcdTM">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>240</y>
        <width>61</width>
        <height>23</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="midLineWidth">
       <number>-1</number>
      </property>
      <property name="smallDecimalPoint">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="lT1">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>35</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>T1</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="lT2">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>70</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>T2</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="lT3">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>110</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>T3</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="lT4">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>150</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>T4</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="lT5">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>190</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>T5</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="lMedia">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>240</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>Media</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="lDesor">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>141</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Desordenados:</string>
      </property>
     </widget>
     <widget class="QLabel" name="lOrd">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>10</y>
        <width>141</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Ordenados:</string>
      </property>
     </widget>
     <widget class="QLabel" name="lpathN">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>240</y>
        <width>271</width>
        <height>21</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>Abrir...</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QToolButton" name="tbSave">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>240</y>
        <width>28</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Load</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>resources/Save_black-128.png</normaloff>resources/Save_black-128.png</iconset>
      </property>
     </widget>
     <widget class="QToolButton" name="tbClear">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>240</y>
        <width>28</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Load</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>resources/new-128.png</normaloff>resources/new-128.png</iconset>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tImagenes">
     <attribute name="title">
      <string>Imagenes</string>
     </attribute>
     <widget class="QGraphicsView" name="gvNormal">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>311</width>
        <height>201</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
     </widget>
     <widget class="QGraphicsView" name="gvTransformada">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>31</y>
        <width>301</width>
        <height>201</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
     </widget>
     <widget class="QToolButton" name="tbLoadI">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>240</y>
        <width>28</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>resources/folder_2_open-512.png</normaloff>resources/folder_2_open-512.png</iconset>
      </property>
     </widget>
     <widget class="QLabel" name="lMedia_3">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>240</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>Media</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="lT5_I">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>190</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>T5</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcdT1I">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>30</y>
        <width>61</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcdT2I">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>70</y>
        <width>61</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="lT2_I">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>70</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>T2</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcdT3I">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>110</y>
        <width>61</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcdT5I">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>190</y>
        <width>61</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcdTMI">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>240</y>
        <width>61</width>
        <height>23</height>
       </rect>
      </property>
      <property name="smallDecimalPoint">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="lT1_I">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>35</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>T1</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcdT4I">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>150</y>
        <width>61</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="lT3_I">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>110</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>T3</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="lT4_I">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>150</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>T4</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QToolButton" name="tbRunI">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>240</y>
        <width>28</width>
        <height>22</height>
       </rect>
      </property>
      <property name="inputMethodHints">
       <set>Qt::ImhNone</set>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>resources/play.png</normaloff>resources/play.png</iconset>
      </property>
     </widget>
     <widget class="QLabel" name="lpathI">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>240</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>Abrir...</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QToolButton" name="tbSave_2">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>240</y>
        <width>28</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Load</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>resources/Save_black-128.png</normaloff>resources/Save_black-128.png</iconset>
      </property>
     </widget>
     <widget class="QLabel" name="lpathI_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>Normal:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="lpathI_3">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>10</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>Blanco y negro:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="bExit">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>320</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Salir</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
